static/src/javascripts/bootstraps/enhanced/liveblog.js

```diff
@@ -70,11 +70,13 @@ const cleanId = (blockIdString) => {
 };
 
 const componentEvent = (pinnedBlockId, action) => ({
-	component: {
-		componentType: 'LIVE_BLOG_PINNED_POST',
-		id: pinnedBlockId,
-	},
-	action: action,
+    componentEvent: {
```

Comment:
this makes a lot of sense

Reply:
Yeah, good spot @abeddow91 . Looking [here](https://github.com/guardian/frontend/blob/main/static/src/javascripts/projects/common/modules/identity/sign-in-gate/component-event-tracking.js#L7) it seems that this extra object is expected. Although, I've also seen examples in the codebase where this isn't used but they could be outdated older code

Reply:
An example of the wrapper not being used [is here](https://github.com/guardian/frontend/blob/a243c455283ecbcc4808dca894522fac2fa6349d/static/src/javascripts/projects/commercial/modules/brazeBanner.js#L121). This code is fairly recent 👀

Reply:
Yeah its a bit confusing! As you said, a platform API would definitely help reduce the error risk!

Reply:
That's odd because that "braze-sdk-load-timing" component isn't a valid component type (or at least not listed in here https://github.com/guardian/ophan/blob/main/event-model/src/main/thrift/componentevent.thrift) so maybe recoding an object of that shape does something else entirely?

Reply:
I wonder if these braze requests just ended up in a different place (as @abeddow91 mentioned in chat) and the Big Query sql was just adjusted for this.

It's actually really useful that this PR was raised as we now know that there is a right way and a not so right way to make these requests and any future DCR work should learn from this

I'll post a link to this thread in the Trello ticket.